x-pack/plugins/observability_solution/observability_ai_assistant_management/public/app.tsx

```diff
@@ -22,9 +23,10 @@ import { AppContextProvider } from './context/app_context';
 interface MountParams {
   core: CoreSetup<StartDependencies, AiAssistantManagementObservabilityPluginStart>;
   mountParams: ManagementAppMountParams;
+  buildFlavor: BuildFlavor;
```

Comment:
Can we use something like `flags: { isSpacesAvailable }` perhaps? spaces will be available on serverless at some point, and preferably we capture that in a single place.